Polyline_simplification_2/demo/Polyline_simplification_2/Polyline_simplification_2.ui

```diff
@@ -1,268 +1,221 @@
-<ui version="4.0" >
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
  <author>GeometryFactory</author>
  <class>Polyline_simplification_2</class>
- <widget class="QMainWindow" name="Polyline_simplification_2" >
-  <property name="geometry" >
+ <widget class="QMainWindow" name="Polyline_simplification_2">
+  <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>600</height>
    </rect>
   </property>
-  <property name="windowTitle" >
+  <property name="windowTitle">
    <string>CGAL Polyline Simplification</string>
   </property>
-  <property name="windowIcon" >
-   <iconset resource="../resources/CGAL.qrc" >
+  <property name="windowIcon">
+   <iconset resource="../../../../CGAL/GraphicsView/demo/resources/CGAL.qrc">
```

Comment:
How can this be a valid path?

Reply:
It's probably only valid on my current config, indeed. Still better than the current path that leads to a file that doesn't exist :)

Reply:
At least it you remove the inner `../CGAL/` it would work in all Git-layouts.